tests/jiva/snapshot/provision_test.go

```diff
@@ -19,6 +19,7 @@ package snapshot
 import (
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+	. "github.com/openebs/maya/tests/jiva"
```

Comment:
Let us not have dot import for this line

Reply:
Done